lib/update-fetch.js

```diff
@@ -127,7 +127,7 @@ exportables.fetchBuild = function(build) {
 
 exportables.downloadTgz = function(tgzUrl, fileMap) {
   return new Promise((resolve, reject) => {
-    log.info('Downloading files. This may take a few minutes...');
+    log.info('Downloading files (this may take a few minutes)...');
```

Comment:
If you deleted the `(this may take a few minutes)` in the squashfs write (which I think is fair now that there is a progress bar), you might as well delete it here too.